src/app/shared/components/cluster-summary/style.scss

```diff
@@ -15,6 +15,8 @@
 @use 'variables';
 @use 'mixins';
 
```

Comment:
Addin comment 

Ticket point is not catered

![single-column layout](https://user-images.githubusercontent.com/17727069/201313802-90929eeb-04ed-4518-a1af-f8b293ae787e.png)